include/tvm/runtime/container/optional.h

```diff
@@ -93,6 +93,11 @@ class Optional : public ObjectRef {
     ICHECK(data_ != nullptr);
     return T(data_);
   }
+  /*!
+   * \return The internal object pointer with type T* for Optional<T>.
+   * \note This function do not perform not-null checking.
+   */
+  const T* get() const { return data_; }
```

Comment:
The type of `T` here is an ObjectRef, instead we should use `ContainerType` as defined a few lines above

Reply:
Thanks! yep, I just noticed the the type difference.